CHANGELOG.md

```diff
@@ -1,5 +1,55 @@
 # Changelog
 
+### Added
+- Added debug endpoints (#1792)
+- Added and improved documentation (#1806, #1808, #1813, #1821, #1831, #1837, #1838, #1861, #1868, #1869, #1876, #1880, #1883, #1886, #1902, #1918, #1923, #1939, #1951, #1970, #1999, #2011)
+- Added improved testing (#1837, #1872, #1873, #1931, #1937, #1942, #1948, #1954, #1958, #1978, #1982)
```

Comment:
I'm not sure the merits of including this line and the next in the changelog. It's nice/reassuring to know that this work is happening, but is it important to consumers of the changelog? Particularly with the documentation line, it's hard to know exactly _what_ improved